packages/components/src/components/drag-and-drop-file-container/src/index.tsx

```diff
@@ -96,14 +98,14 @@ const DragAndDropFileContainer = React.forwardRef<
               </Box>
             )}
 
-            <Text css={{ marginBottom: "$35" }}>
-              {isDragReject ? rejectText : uploadText}
-            </Text>
+            <Box css={{ marginBottom: "$35" }}>
+              <Text>{isDragReject ? rejectText : uploadText}</Text>
+            </Box>
 
             <Button
               variant="secondary"
               onClick={open}
-              css={
+              style={
                 isDragAccept
                   ? {
```

Comment:
Similar here. We can always use `style` if we need to escape the system and override a component style.